src/Core/src/Handlers/SwipeView/SwipeViewHandler.Windows.cs

```diff
@@ -0,0 +1,204 @@
+using System;
+using System.Linq;
+using WSwipeControl = Microsoft.UI.Xaml.Controls.SwipeControl;
+using WSwipeItems = Microsoft.UI.Xaml.Controls.SwipeItems;
+using WSwipeItem = Microsoft.UI.Xaml.Controls.SwipeItem;
+using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class SwipeViewHandler : ViewHandler<ISwipeView, WSwipeControl>
+	{
+		protected override WSwipeControl CreateNativeView() => new();
+
+		public override void SetVirtualView(IView view)
+		{
+			base.SetVirtualView(view);
+			_ = VirtualView ?? throw new InvalidOperationException($"{nameof(VirtualView)} should have been set by base class.");
+			_ = NativeView ?? throw new InvalidOperationException($"{nameof(NativeView)} should have been set by base class.");
+		}
+
+		public static void MapContent(ISwipeViewHandler handler, ISwipeView view)
+		{
+			_ = handler.NativeView ?? throw new InvalidOperationException($"{nameof(NativeView)} should have been set by base class.");
+			_ = handler.MauiContext ?? throw new InvalidOperationException($"{nameof(MauiContext)} should have been set by base class.");
+			_ = handler.TypedVirtualView ?? throw new InvalidOperationException($"{nameof(VirtualView)} should have been set by base class.");
+
+
+			if (handler.TypedVirtualView.PresentedContent is not IView presentedView)
+				return;
+
+			handler.TypedNativeView.Content = presentedView.ToNative(handler.MauiContext, true);
+		}
+
+		public static void MapSwipeTransitionMode(ISwipeViewHandler handler, ISwipeView swipeView)
+		{
+		}
+
+		public static void MapRequestOpen(ISwipeViewHandler handler, ISwipeView swipeView, object? args)
+		{
+			if (args is not SwipeViewOpenRequest)
+			{
+				return;
+			}
+		}
+
+		public static void MapRequestClose(ISwipeViewHandler handler, ISwipeView swipeView, object? args)
+		{
+			handler.TypedNativeView.Close();
+		}
+
+
+		protected override void ConnectHandler(WSwipeControl nativeView)
+		{
+			base.ConnectHandler(nativeView);
+			NativeView.Loaded += OnLoaded;
+		}
+
+		protected override void DisconnectHandler(WSwipeControl nativeView)
+		{
+			base.DisconnectHandler(nativeView);
+			NativeView.Loaded -= OnLoaded;
+		}
+
+		void OnLoaded(object sender, UI.Xaml.RoutedEventArgs e)
+		{
+			if (!NativeView.IsLoaded)
+				return;
+
+			UpdateValue(nameof(ISwipeView.LeftItems));
```

Comment:
I think this is to avoid an exception with the SwipeControl, right?. Could you include a comment here to remember that in the future.